### ISSUE TITLE
Adding Ticket Authors to Watchers

### ISSUE THREAD

**Issue by shawnpk** [rubysherpas/active_rails_examples#38]
Chapter 11 deals with sending emails and adding watchers. Upon finishing this chapter, I noticed that we don't actually add the ticket author to the watchers. In `spec/features/ticket_notifications_spec.rb`, we assign 'Alice' directly to the `ticket.watchers', but I don' see anywhere in the code where we actually assign the ticket author to the watchers.

This is in the version generated on 2021-02-12 from 0a83c5577 in the epub version.

**Comment by radar** [comment]
You're right! This should really be added before the "Previewing emails" section in Chapter 11. I'll leave this open so we can address it next time we run through the book.
